src/libraries/System.Private.CoreLib/src/System/Obsoletions.cs

```diff
@@ -9,5 +9,14 @@ internal static class Obsoletions
 
         internal const string SystemTextEncodingUTF7Message = "The UTF-7 encoding is insecure and should not be used. Consider using UTF-8 instead.";
         internal const string SystemTextEncodingUTF7DiagId = "MSLIB0001";
+
+        internal const string ConstrainedExecutionRegionMessage = "The Constrained Execution Region (CER) feature is no longer implemented.";
+        internal const string ConstrainedExecutionRegionDiagId = "MSLIB0004";
+
+        internal const string GlobalAssemblyCacheMessage = "The Global Assembly Cache is no longer implemented.";
+        internal const string GlobalAssemblyCacheDiagId = "MSLIB0005";
+
+        internal const string ThreadAbortMessage = "Thread.Abort is no longer supported and throws PlatformNotSupportedException.";
```

Comment:
This says "no longer supported" and the other ones say "no longer implemented". Should we say "no longer supported" everywhere?

The docs usually say "supported". E.g. https://docs.microsoft.com/en-us/dotnet/framework/performance/constrained-execution-regions